templates/scaffold/typescript/express/README.md

```diff
@@ -61,25 +63,25 @@ This model types can be modified to add new fields to the document.
 
 ### Routes
 
-The `src/controllers` directory contains classes which setup [Express](https://expressjs.com) Web framework CRUD routes for every collection model.
+The `src/routes` directory contains classes which setup [Express](https://expressjs.com) Web framework CRUD routes for every collection model.
 Once project is started, they can be tested using curl commands.
 {{if gt (len .Collections) 0}}
 For example:
 {{with (index .Collections 0)}}
 #### Create document in the `{{.JSON}}` collection:
 ```
-curl -X POST "localhost:8080/{{.JSON}}" -H 'Content-Type: application/json' 
+curl -X POST "localhost:3000/{{.JSON}}" -H 'Content-Type: application/json' 
```

Comment:
Can you add a complete example here with the task object to be inserted

Reply:
I cannot. It's generic, we would need functionality to generate synthetic document from the collection schema.
Which would be cool and we could generate integration tests automatically.
I couldn't find any library doing that, implementing it would take some time.

Reply:
Makes sende